src/sidemenu/PinnedMenu.tsx

```diff
@@ -33,22 +33,41 @@ const PinnedMenuItem: React.FC<
   const { item, current, menuOptions, label } = props as PinnedMenuItemProps;
   const { loading, loadingAccentColor } = props as LoadingPinnedMenuItemProps;
 
+  const getNotificationlabel = (item: BoardType | CircleButtonProps, label?: string) => {
+    if (!label) {return ""}
```

Comment:
If you use `{` the return should go in its own line. I thought prettier would fix this?

Reply:
Making another comment here cause i think it won't be picked up as a review comment if I made the previous comment not during the review? I'm still learning GitHub too.

Reply:
Turns out prettier doesn't work on save if you have Autosave After Delay on. I have now switched it to Save on Focus Change, so hopefully prettier will actually catch these little things.